consensus/src/consensus/mod.rs

```diff
@@ -696,6 +700,120 @@ impl ConsensusApi for Consensus {
         sample_headers
     }
 
+    fn get_utxo_return_address(&self, txid: Hash, target_daa_score: u64) -> ReturnAddress {
```

Comment:
I think we should consider moving this logic to a sub manager/processor or at least to another module file. Wdyt?

Reply:
Did you have a specific location in mind?

Reply:
virtual processor probably makes sense

Reply:
Moved in [b6b6ec1](https://github.com/kaspanet/rusty-kaspa/pull/436/commits/b6b6ec16c932f9abb734e5e8cfa015698213c9b1)